atlasdb-commons/src/main/java/com/palantir/common/base/BatchingVisitableView.java

```diff
@@ -191,8 +191,8 @@ public T getMin(final Ordering<? super T> o, @Nullable T defaultElement) {
     }
 
     /**
-     * Returns an immutable copy of the current contents of this iterable view. Does not support
-     * null elements.
+     * Returns an immutable List of all the objects in the iterable view (not a single batch).
+     * Does not support null elements.
```

Comment:
Chat discussion:

> @SerialVelocity
> I've been thinking more about the `immutableCopy()` JavaDoc and I'm thinking it's super weird to add extra text saying that the immutable copy contains the contents of all the batches. Everything in `BatchingVisitableView` only allows you to interact with the full batch (e.g. `find`, `copyInto`, `any`, `transform`, `forEach`, etc)
> It's not a view of a single batch. It's a view that allows you to visit the contents in batches.
> 
> @hsaraogi 
> yup.. every method deals with all contents, but it was confusing to our users.. I believed it was getRange returning an object with “Batch” in the name..
> We could add a class JavaDoc saying that BatchingVisitable operates on all the batches..
> 
> @SerialVelocity
> I think JavaDoc on the `BatchingVisitable` or `BatchingVisitableView` class would be better.
> Since I have a feeling that if we change the JavaDoc on `immutableCopy`, people will say in the future, "Oh, I thought `copyInto` would only copy the batch since it explicitly states in `immutableCopy` that it copies everything"

Blocked on checking with users what would alleviate this pain.